### PR TITLE
Timestamps on created and modified at values

### PR DESCRIPTION
**Pull request by luka-nextcloud**
Signed-off-by: Luka Trovic <luka@nextcloud.com>

* Resolves: #2818
* Target version: master 

### Checklist

- [x] Code is properly formatted
- [x] Sign-off message is added to all commits
- [ ] Tests (unit, integration, api and/or acceptance) are included
- [x] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: src/components/card/CardSidebar.vue [hunk 1]
```diff
@@ -25,6 +25,7 @@
 		:active="tabId"
 		:title="title"
 		:subtitle="subtitle"
+		:subtitleTooltip="subtitleTooltip"
```
Comment (juliushaertl) [inline src/components/card/CardSidebar.vue:28]: Mind to fix the eslint warning?

src/components/card/CardSidebar.vue#L28
Attribute ':subtitleTooltip' must be hyphenated

**Comment by juliushaertl** [comment]
Otherwise looks good, LGTM once linter is happy :+1:

File: src/components/card/CardSidebar.vue [hunk 2]
```diff
@@ -142,6 +146,9 @@ export default {
 		subtitle() {
 			return t('deck', 'Modified') + ': ' + this.relativeDate(this.currentCard.lastModified * 1000) + ' ' + t('deck', 'Created') + ': ' + this.relativeDate(this.currentCard.createdAt * 1000)
 		},
+		subtitleTooltip() {
+			return t('deck', 'Modified') + ': ' + this.formatDate(new Date(this.currentCard.lastModified * 1000)) + '\n' + t('deck', 'Created') + ': ' + this.formatDate(new Date(this.currentCard.createdAt * 1000))
```
Comment (eneiluj) [inline src/components/card/CardSidebar.vue:150]:
```suggestion
			return t('deck', 'Modified') + ': ' + this.formatDate(this.currentCard.lastModified) + '\n' + t('deck', 'Created') + ': ' + this.formatDate(this.currentCard.createdAt)
```

File: src/components/card/CardSidebar.vue [hunk 3]
```diff
@@ -189,6 +196,9 @@ export default {
 		closeModal() {
 			this.$store.dispatch('setConfig', { cardDetailsInModal: false })
 		},
+		formatDate(date) {
+			return moment(date).locale(this.locale).format('LLLL')
```
Comment (eneiluj) [inline src/components/card/CardSidebar.vue:200]:
```suggestion
		formatDate(timestamp) {
			return moment.unix(timestamp).locale(this.locale).format('LLLL')
```

**Review by eneiluj — CHANGES_REQUESTED** [review]
This is not a big difference but I would prefer passing the timestamp directly to `moment` instead of calculating the milliseconds and building a Date object.

What do you think?

**Review by eneiluj — APPROVED** [review]

**Pull request merged**